Framework/PythonInterface/test/python/mantid/SimpleAPITest.py

```diff
@@ -27,6 +27,20 @@ def test_python_algorithms_are_loaded_recursively(self):
     def test_version_number_equals_2(self):
         self.assertEqual(simpleapi.apiVersion(), 2)
 
+    def test_simpleapi_does_not_import_qt(self):
```

Comment:
I love these tests!

Reply:
I wonder if we could slightly improve the output from when they fail though? For some developers it may not be entirely obvious what the problem is. Can we add a [`msg`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertFalse) parameter to give some indication to developers of what  has gone wrong and what to look for in terms of fixing it?
My main concern is that it may not be an obvious `import` in the framework but a more circuitious route like what you have fixed here with the engineering scripts and developers are not sure where to start.